shared/pingpong/pingpong.go

```diff
@@ -410,19 +463,16 @@ func sendFromTo(fromList, toList []string, accounts map[string]uint64, assetPara
 			sendErr = client.BroadcastTransactionGroup(stxGroup)
 		}
 
-		if sendErr != nil && !cfg.Quiet {
-			_, _ = fmt.Fprintf(os.Stderr, "error sending transaction: %v\n", sendErr)
-		} else {
```

Comment:
removed `if sendErr != nil && !cfg.Quiet {` branch since `!cfg.Quiet` does not make much sense because of outputting the same error below under `if sendErr != nil {`